docs/source/data_analysis/HPC-module-SLEAP.md

```diff
@@ -0,0 +1,610 @@
+4043603       slp_infer        gpu     FAILED 2023-10-10T18:27:32   00:01:37
+4043603.bat+      batch                FAILED 2023-10-10T18:27:32   00:01:37    423476K
+4043611       slp_infer        gpu    PENDING             Unknown   00:00:00
+```
+:::
+
+:::{tab-item} view the logs
+
+View the contents of standard output and error
+(the node name and job ID will differ in each case):
+```{code-block} console
+$ cat slurm.gpu-sr670-20.3445652.out
+$ cat slurm.gpu-sr670-20.3445652.err
+```
+:::
+
+::::
+
+```{dropdown} Out-of-memory (OOM) errors
+:color: warning
+:icon: alert-fill
+
+If you encounter out-of-memory errors, keep in mind that there two main sources of memory usage:
+- CPU memory (RAM), specified via the `--mem` argument in the SLURM batch script. This is the memory used by the Python process running the training job and is shared among all the CPU cores.
+- GPU memory, this is the memory used by the GPU card(s) and depends on the GPU card type you requested via the `--gres gpu:1` argument in the SLURM batch script. To increase it, you can request a specific GPU card type with more GPU memory (e.g. `--gres gpu:a4500:1`). The SWC wiki provides a [list of all GPU card types and their specifications](https://wiki.ucl.ac.uk/display/SSC/CPU+and+GPU+Platform+architecture).
+- If requesting more memory doesn't help, you can try reducing the size of your SLEAP models. You may tweak the model backbone architecture, or play with *Input scalng*, *Max stride* and *Batch size*. See SLEAP's [documentation](https://sleap.ai/) and [discussion forum](https://github.com/talmolab/sleap/discussions) for more details.
+```
+
+### Evaluate the trained models
+Upon successful completion of the training job, a `models` folder will have
+been created in the training job directory. It contains one subfolder per
+training run (by default prefixed with the date and time of the run).
+
+```{code-block} console
+$ cd /ceph/scratch/neuroinformatics-dropoff/SLEAP_HPC_test_data
+$ cd labels.v001.slp.training_job
+$ cd models
+$ ls -1
+230509_141357.centered_instance
+230509_141357.centroid
+```
+
+Each subfolder holds the trained model files (e.g. `best_model.h5`),
+their configurations (`training_config.json`) and some evaluation metrics.
+
+```{code-block} console
+$ cd 230509_141357.centered_instance
+$ ls -1
+best_model.h5
+initial_config.json
+labels_gt.train.slp
+labels_gt.val.slp
+labels_pr.train.slp
+labels_pr.val.slp
+metrics.train.npz
+metrics.val.npz
+training_config.json
+training_log.csv
+```
+The SLEAP GUI on your local machine can be used to quickly evaluate the trained models.
+
+- Select *Predict* -> *Evaluation Metrics for Trained Models...*
+- Click on *Add Trained Models(s)* and select the folder containing the model(s) you want to evaluate.
+- You can view the basic metrics on the shown table or you can also view a more detailed report (including plots) by clicking *View Metrics*.
```

Comment:
Maybe also mention the [model evaluation notebook](https://sleap.ai/notebooks/Model_evaluation.html) from SLEAP

Reply:
sweet, will do